presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConfig.java

```diff
@@ -153,4 +160,95 @@ public KafkaConfig setMessagesPerSplit(int messagesPerSplit)
         this.messagesPerSplit = messagesPerSplit;
         return this;
     }
+
+    @Config("kafka.security.protocol")
+    @ConfigDescription("Security protocol used for Kafka connection.")
+    public KafkaConfig setSecurityProtocol(String securityProtocol)
+    {
+        this.securityProtocol = securityProtocol;
+        return this;
+    }
+
+    public String getSecurityProtocol()
+    {
+        return securityProtocol;
+    }
+
+    @Config("kafka.ssl.truststore.location")
```

Comment:
We should not allow user to configure this properties when security protocol is set to plain text.

Reply:
Will have a check on security.protocol when this is configured